www/base/src/app/builders/builders.tpl.jade

```diff
@@ -37,7 +39,10 @@
             span(ng-show="tags_filter.length > 0")
                 span.label.clickable.label-danger(ng-click="tags_filter = []") x
           th(style="width:20%px;") Workers
-      tr(ng-repeat='builder in builders | filter: isBuilderFiltered | orderBy: "name"')
+      ul(uib-pagination, total-items="builders.length",
```

Comment:
total-items is not builders.length, it has to be the number of builders filtered. Else, you will have empty pages in the end, like mojca suggested

Reply:
Ok, fixed that now